controllers/cpms/cpms_controller.go

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright 2022.
```

Comment:
Should this be updated at some point?  I suppose it's autogenerated with 2022?

Reply:
ah yes. I just copied it from another file. Let me update to 2024